janus/lib/models/token.rb

```diff
@@ -123,6 +125,9 @@ def valid_task_token?(janus_user)
       return false if project_name == 'administration'
       return false if token_role == 'administrator'
 
+      # if the user is a supereditor, the request is honored
```

Comment:
This and related changes introduce the ability for a supereditor (or superuser) to make a task token for any project without having to be a member of that project. Ergo, Polyphemus has to be a supereditor now (i.e., in the administration group), but no longer has to be licensed to specific projects in order to run their jobs.

Reply:
At some point, we should consider refresh tokens that are still authorized and scoped by individual users.  My concern is the way that super editor contains the unique privilege of not only having access to everything, but effectively impersonating anyone as well.  That said, this works for now for sure.  Before we get to the point of refresh tokens we'll need to think the executor approach anyways.

Reply:
Are we still giving super-editor privileges to non-DL folks? At some point I thought we gave Vincent some elevated privileges...?